### PR TITLE
multi_queues_test: Add params for sub_test 'multi_vms_nics'

### PR DESCRIPTION
**Pull request by sitoliu**
ID: 1647296
Signed-off-by: Sitong Liu <siliu@redhat.com>

### CODE REVIEW

**Comment by sitoliu** [comment]
@vivianQizhu update the commit, please help check again, thanks.

> When run bg_sub_test, it will run script multi_vms_nics.py directly,
so params needed in multi_vms_nics.py with 'params.get()' should be 
included in main test's cfg file multi_queues_test.cfg.

File: generic/tests/cfg/multi_queues_test.cfg [hunk 1]
```diff
@@ -94,3 +94,7 @@
             nics += " nic2"
             vms += " vm2"
             image_snapshot = yes
+            check_irqbalance_cmd = "service irqbalance status"
```
Comment (hereischen) [inline generic/tests/cfg/multi_queues_test.cfg]: I think it is better provide the default value in the `multi_vms_nics.py`'s`params.get()`.
Eg. `params.get("check_irqbalance_cmd", "service irqbalance status")`

Comment (vivianQizhu) [reply generic/tests/cfg/multi_queues_test.cfg]: Agree with @hereischen , but recommend use `systemctl` as default.

Comment (sitoliu) [reply generic/tests/cfg/multi_queues_test.cfg]: Thanks, update it.

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

**Comment by vivianQizhu** [comment]
@xiywang Please help review, thanks.

**Comment by xiywang** [comment]
LGTM.
Acked-by: Xiyue Wang xiywang@redhat.com

**Pull request merged**